pkg/recipes/terraform/config/providers/aws_test.go

```diff
@@ -33,8 +35,11 @@ import (
 var (
 	testRegion         = "test-region"
 	testAWSCredentials = ucp_credentials.AWSCredential{
-		AccessKeyID:     "testAccessKey",
-		SecretAccessKey: "testSecretKey",
+		Kind: ucp_datamodel.AWSAccessKeyCredentialKind,
+		AccessKeyCredential: &ucp_datamodel.AWSAccessKeyCredentialProperties{
+			AccessKeyID:     "testAccessKey",
+			SecretAccessKey: "testSecretKey",
+		},
 	}
```

Comment:
Should we have IRSACredential unit tests?

Reply:
Yes, I going to add them as part of Terraform RP changes.